modules/tinymce/src/plugins/lists/test/ts/browser/ContentEditableFalseActionsTest.ts

```diff
@@ -127,13 +127,14 @@ ${listContent}
     })
   );
 
-  it('TINY-9458: InsertOrderedList command should noop if noneditable blocks are selected', () => {
+  it('TINY-9823: InsertOrderedList command should ignore noneditable blocks', () => {
     const editor = hook.editor();
     const initialContent = '<p>a</p>\n<p contenteditable="false">b</p>\n<p>c</p>';
+    const expectedContent = '<ol>\n<li>a</li>\n</ol>\n<p contenteditable="false">b</p>\n<ol>\n<li>c</li>\n</ol>';
```

Comment:
Is this the result we'd expect? We're now making two lists, aren't we? I'd have expected it to 'lift' the CEF element into a li wholesales, not split around it.

Reply:
Maybe we can improve this later having it split like this makes the fix simpler. Wrapping the cefs block in LIs would require some special code maybe some step we do before converting to lists.